doc/user.rst

```diff
@@ -91,7 +90,11 @@ For example, the following installs a fairly complete list of build dependencies
       libsystemd-dev \
       libgtest-dev \
       libboost-dev \
-      libicu-dev
+      libicu-dev \
+      libchromaprint-dev \
+      libgcrypt20-dev \
+      python3-sphinx \
```

Comment:
Sphinx isn't needed to build MPD. And since its dependencies are very heavy, I don't want to make everybody install it without a good reason.